app/src/main/res/layout/fragment_splash.xml

```diff
@@ -17,10 +17,12 @@
 
             <ImageView
                 android:id="@+id/circle"
-                android:layout_width="120dp"
-                android:layout_height="120dp"
-                android:layout_marginTop="24dp"
-                android:src="@drawable/splash_circle"
+                android:layout_width="80dp"
```

Comment:
How safe is it to use specific dims (as opposed to relative dims) for responsive things? I've been playing it by ear when to hard code vs relative, wonder if you have any rules of thumb

Reply:
Not really. However, relative distances with fixed sizes are more problematic.
ConstraintLayout does allow to use relative sizes but it requires some more work to set it up correctly. So, at least for now, I prefer to stick with the ScrollView which guarantees that you will be able to reach all elements on all screens/orientations.

Reply:
You can use `app:layout_constraintDimensionRatio` to set specific ratio and get rid of hardcoded sizes

Reply:
Hmm, nice! Will try it!